pkg/log/log.go

```diff
@@ -46,23 +48,50 @@ func WithContext(ctx context.Context) {
 	Info().WithContext(ctx)
 }
 
+func WithError(err error) {
+	Error().WithError(err)
+}
+
 func (l *logger) Print(msg string) {
+	logFields := make(logrus.Fields)
+	if ctxFields := field.FromContext(l.ctx); ctxFields != nil {
```

Comment:
```suggestion
	if ctxFields := field.FromContext(l.ctx); len(ctxFields) != 0 {
```

Reply:
@tdmanv ctxFields is of type field.Fieldser interface. I think `len(ctxFields)` will fail.

Reply:
you're right! please ignore this comment